src/__tests__/array/array.ts

```diff
@@ -70,4 +71,29 @@ export type TestList = Test[];
  */
 export type TestList = string[];`);
   });
+
+  test('test to ensure sparse arrays have undefined as a possible type', () => {
+    // this tests this code

```

Comment:
Can this be removed?

Reply:
@mrjono1 - Yep! Do you want me to remove the test and comment or just the comment?

Reply:
@mrjono1 - bump

Reply:
Remove the commented out code please

Reply:
@mrjono1 - Done